src/tracing/service/tracing_service_impl.cc

```diff
@@ -4097,7 +4142,8 @@ size_t TracingServiceImpl::PurgeExpiredAndCountTriggerInWindow(
 
 base::Status TracingServiceImpl::FlushAndCloneSession(
     ConsumerEndpointImpl* consumer,
-    ConsumerEndpoint::CloneSessionArgs args) {
+    ConsumerEndpoint::CloneSessionArgs args,
```

Comment:
did it really not occur to you, after having added this "fd argument" dozen times "hmm maybe I should put it in CSArgs?"